zebra-consensus/src/chain/tests.rs

```diff
@@ -102,6 +104,86 @@ fn verifiers_from_network(
     verifiers_from_checkpoint_list(network, CheckpointList::new(network))
 }
 
+static BLOCK_VERIFY_TRANSCRIPT_GENESIS: Lazy<
+    Vec<(Arc<Block>, Result<BlockHeaderHash, TransError>)>,
+> = Lazy::new(|| {
+    let block: Arc<_> =
+        Block::zcash_deserialize(&zebra_test::vectors::BLOCK_MAINNET_GENESIS_BYTES[..])
+            .unwrap()
+            .into();
+    let hash = Ok(block.as_ref().into());
+
+    vec![(block, hash)]
+});
```

Comment:
Is there a reason to keep these as statics? @ZcashFoundation/zebra-team 

I think the readability issues @teor2345 brought up might be helped by just inlining the content of these setup statics into the tests that use them if possible.

Reply:
Not sure about this instance but they're defined like that elsewhere so that you can have a list of transcripts and have a single test that iterates over each transcript to keep all the common code the same.